src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -262,10 +263,13 @@ uct_rc_mlx5_iface_handle_failure(uct_ib_iface_t *ib_iface, void *arg,
 
     status  = uct_iface_handle_ep_err(&iface->super.super.super,
                                       &ep->super.super.super, ep_status);
+
     log_lvl = uct_base_iface_failure_log_level(&ib_iface->super, status,
                                                ep_status);
 
     uct_ib_mlx5_completion_with_err(ib_iface, arg, &ep->tx.wq, log_lvl);
+    ucs_arbiter_dispatch(&iface->tx.arbiter, 1, uct_rc_ep_process_pending,
+                         NULL);
```

Comment:
why needed here? It is called in  `uct_rc_mlx5_iface_update_tx_res`

Reply:
we removed it from `uct_rc_mlx5_iface_update_tx_res`